### PR TITLE
Check restart of fail2ban

### PR DESCRIPTION
**Pull request by maniackcrudelis**
Check the log to see if the jail for this app is correctly loaded.

### CODE REVIEW

File: ynh_add_fail2ban_config/ynh_add_fail2ban_config [hunk 1]
```diff
@@ -28,21 +28,26 @@ enabled = true
...
 EOF
 
 	ynh_store_file_checksum "$finalfail2banjailconf"
 	ynh_store_file_checksum "$finalfail2banfilterconf"
   
 	sudo systemctl restart fail2ban
+	if local fail2ban_error="$(tail -n50 /var/log/fail2ban.log | grep "WARNING Command.*$app.*addfailregex")"
+	then
+		echo "[ERR] Fail2ban failed to load the jail for $app" >&2
+		echo "WARNING${fail2ban_error#*WARNING}" >&2
```
Comment (JimboJoe) [inline ynh_add_fail2ban_config/ynh_add_fail2ban_config]: What is this command supposed to do?

Comment (maniackcrudelis) [reply ynh_add_fail2ban_config/ynh_add_fail2ban_config]: Hum, the first line grep an error in the log file.
And, if grep find something, it simply print this part of the log.

Comment (JimboJoe) [reply ynh_add_fail2ban_config/ynh_add_fail2ban_config]: Hmm thanks 😆 
I was referring to line 49 specifically... 😉

Comment (maniackcrudelis) [reply ynh_add_fail2ban_config/ynh_add_fail2ban_config]: Oh ok;)
Sorry that true that I forgot to comment my code! Shame on me...
This print removes the beginning of the line which is essentially a timestamp and other boring stuff

File: ynh_add_fail2ban_config/ynh_add_fail2ban_config [hunk 2]
```diff
@@ -42,9 +42,10 @@ EOF
   ynh_store_file_checksum "$finalfail2banjailconf"
   ynh_store_file_checksum "$finalfail2banfilterconf"
   
-..sudo systemctl restart fail2ban
-..if local fail2ban_error="$(journalctl -u fail2ban | tail -n50 | grep "WARNING Command.*$app.*addfailregex")"
-..then
+  systemctl restart fail2ban
+  local fail2ban_error="$(journalctl -u fail2ban | tail -n50 | grep "WARNING.*$app.*")"
```
Comment (maniackcrudelis) [inline ynh_add_fail2ban_config/ynh_add_fail2ban_config]: Why did you removed "Command." and ".*addfailregex" for the grep command ?

Comment (JimboJoe) [reply ynh_add_fail2ban_config/ynh_add_fail2ban_config]: I had other WARNING that were interesting to show (among with the `regrexp` typo 😄)... ideally there shouldn't be any remaining warnings.

**Pull request merged**